### PR TITLE
LPS-153371 Update Alloy Editor to v2.14.6

### PR DESCRIPTION
**Pull request by markocikos**
### What is the goal of this PR?

We are updating Alloy Editor to v2.14.6

Running `yarn add alloyeditor@2.14.6` generated `yarn.lock` changes.

### What's Changed
* fix: add 'rel' attribute to the generated links by @orsolyaDekany in https://github.com/liferay/alloy-editor/pull/1522
* fix: #1524 indent and outdent icons are switched by @fortunatomaldonado in https://github.com/liferay/alloy-editor/pull/1525

### New Contributors
* @orsolyaDekany made their first contribution in https://github.com/liferay/alloy-editor/pull/1522

**Full Changelog**: https://github.com/liferay/alloy-editor/compare/v2.14.5...v2.14.6

### CODE REVIEW

File: modules/yarn.lock [hunk 1]
```diff
@@ -2373,10 +2365,10 @@
     "@types/react" "*"
     hoist-non-react-statics "^3.3.0"
 
-"@types/html-minifier-terser@5.1.0", "@types/html-minifier-terser@^6.0.0":
```
Comment (diegonvs) [inline modules/yarn.lock:2376]: Those changes on yarn.lock seems weird. Could you run an `npx yarn-deduplicate` inside `modules/` folder?

Comment (diegonvs) [reply modules/yarn.lock:2376]: You did 

> Running yarn add alloyeditor@2.14.6 generated yarn.lock changes.

What I use to do is:

1. Go to [alloyeditor-web module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-alloyeditor-web/package.json#L3) and change the version here
2. Go to `modules/` and run `yarn install` and then `npx yarn-deduplicate`
3. Go back to `alloyeditor-web` module and `gw clean deploy` there to see if everything is on the right place testing the UI :)

Comment (markocikos) [reply modules/yarn.lock]: Force-pushed changes!

I ran `yarn install` and `npx yarn-deduplicate`, it did rollback several `yarn.lock` changes. I added this to docs, to the new [Updating Alloy Editor in DXP section](https://github.com/liferay/alloy-editor/wiki/Contributing#updating-alloy-editor-in-dxp).

As for UI, the new code is correctly applied, we can make the release.

But! @orsolyaDekany during my testing I noticed there is a potential problem with the `ref` solution. Attribute does get added, but it disappears after save. See gif:

screenshot

Comment (orsolyaDekany) [reply modules/yarn.lock]: Hi @markocikos , I was not able to reproduce it on my side, `rel` attribute is still there after publishing the blog or web content.

I tested the fix locally linking the AlloyEditor repo to Liferay Portal repo.
Is there something in Liferay Portal that removes the attribute? I am looking into it.

Comment (markocikos) [reply modules/yarn.lock]: I don't know, I cannot not reproduce it :slightly_smiling_face: . Maybe it's something weird on my side. Can you ask somebody else to try?

I tried on Chrome 101 and Firefox 100, Ubuntu 20

Comment (orsolyaDekany) [reply modules/yarn.lock]: Okay, we figure out that the issue is not reproducible on 7.3.x and 7.2.x, but it is on Master. Currently, I do not know why is the difference.

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**